src/page/GroupCreate.tsx

```diff
@@ -68,13 +71,39 @@ const GroupCreate = () => {
     setIsActive(allFieldsFilled || (isPublicGroup && allFieldsFilled));
   }, [inputs, isPublicGroup]);
 
-  const handleGroupCreate = () => {
-    navigate('/group-complete', {
-      state: {
-        groupPassword: inputs.secretKey,
-        thumbnailUrl: previewImage as string,
-      },
-    });
+  const handleGroupCreate = async () => {
+    const postData = {
+      title: inputs.title,
+      password: inputs.secretKey,
+      capacity: inputs.num,
+      tags: selectedTags,
+      introduce: inputs.intro,
+      information: inputs.group,
+    };
+    const requestBody = new FormData();
+    const jsonChange = JSON.stringify(postData);
+
+    requestBody.append('request', jsonChange);
+
+    if (selectdImageFile) {
+      requestBody.append('imageFile', selectdImageFile);
+    }
+
+    try {
+      const data = await GroupPost(requestBody);
+      const uuid = data.data.uuid;
```

Comment:
P5: 구조분해할당을 활용하시면 더 깔끔해질 것 같아요!
```suggestion
      const { uuid } = data.data;
```

Reply:
넵 감사합니다 !